snooty/taxonomy.toml

```diff
@@ -4,55 +4,221 @@ name = "reference"
 [[genres]]
 name = "tutorial"
 
+# trying to map to docs "name" property 
+# for target_platforms and sub_platforms
+# as much as possible
 [[target_platforms]]
 name = "atlas"
-[[target_platforms.versions]]
-name = "v1.2"
-[[target_platforms.versions]]
-name = "master"
+
+  [[target_platforms.sub_platforms]]
+  name = "atlas-app-services"
+  display_name = "App Services"
+
+  [[target_platforms.sub_platforms]]
+  name = "charts"
+  display_name = "Charts"
+
+  [[target_platforms.sub_platforms]]
+  name = "atlas-cli"
+  display_name = "Atlas CLI"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-api"
+  display_name = "Data API"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-federation"
+
+  [[target_platforms.sub_platforms]]
+  name = "data-lake"
+
+  [[target_platforms.sub_platforms]]
+  name = "device-sync" #404
+
+  [[target_platforms.sub_platforms]]
+  name = "kubernetes-operator"
+
+  [[target_platforms.sub_platforms]]
+  name = "online-archive"
+
+  [[target_platforms.sub_platforms]]
+  name = "search"
+
+  [[target_platforms.sub_platforms]]
+  name = "stream-processing"
+
+  [[target_platforms.sub_platforms]]
+  name = "triggers"
+
+[[target_platforms]]
+name = "bi-connector"
+display_name = "BI Connector"
+
+[[target_platforms]]
+name = "com"
```

Comment:
I think this one is Cloud Manager, but maybe it's weird because it builds from the same repo as Ops Manager?  Do we need a display_name?

Reply:
good catch! added a Cloud Manager display_name